### PR TITLE
Removed multiSlsDetector dependency from slsDetector

### PR DESCRIPTION
**Pull request by erikfrojdh**
- Removed multiSlsDetector dependency from slsDetector by moving writing of config file to multi.
- Removed static version of freeSharedMemory from slsDetector since all should go through multi

### CODE REVIEW

File: slsDetectorSoftware/src/slsDetector.cpp [hunk 1]
```diff
@@ -948,124 +940,80 @@ int slsDetector::updateDetector() {
...
-        names.emplace_back("rx_hostname");
-
-        names.emplace_back("extsig:0");
-        names.emplace_back("vhighvoltage");
+        base.emplace_back("detectormac");
+        base.emplace_back("detectorip");
+        base.emplace_back("rx_udpport");
+        base.emplace_back("rx_udpip");
+        base.emplace_back("rx_udpmac");
+        base.emplace_back("extsig:0");
```
Comment (thattil) [inline slsDetectorSoftware/src/slsDetector.cpp]: For extsig, we removed the signal index. So, just "extsig" should do.

File: slsDetectorSoftware/src/slsDetector.cpp [hunk 2]
```diff
@@ -948,124 +940,80 @@ int slsDetector::updateDetector() {
...
-        names.emplace_back("rx_hostname");
-
-        names.emplace_back("powerchip");
-        names.emplace_back("vhighvoltage");
+        base.emplace_back("detectormac");
+        base.emplace_back("detectorip");
+        base.emplace_back("rx_udpport");
+        base.emplace_back("rx_udpip");
+        base.emplace_back("rx_udpmac");
+        base.emplace_back("powerchip");
```
Comment (thattil) [inline slsDetectorSoftware/src/slsDetector.cpp]: This is a bug I introduced sometime back. Could you please remove the powerchip command for Moench?

File: slsDetectorSoftware/src/slsDetector.cpp (hunk 1 above)
Comment (erikfrojdh) [reply slsDetectorSoftware/src/slsDetector.cpp]: done

File: slsDetectorSoftware/src/slsDetector.cpp (hunk 2 above)
Comment (erikfrojdh) [reply slsDetectorSoftware/src/slsDetector.cpp]: done

**Pull request merged**